website/docs/cloud-docs/agents/telemetry.mdx

```diff
@@ -80,7 +80,7 @@ the HCP Terraform Agent can talk to) on port 4317 of the local machine.
 The OpenTelemetry collector takes configuration in the form of a YAML file. It
```

Comment:
Suggesting rewriting the following sentences to meet our style guide -> if you like this you'd remove the following lines. Thank you so much!
```suggestion
The OpenTelemetry collector takes configuration in the form of a YAML file. Refer to the [default YAML configuration file](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/config.yaml) as a starting point for writing your own configuration.
```